docs/product/data-management-settings/scrubbing/advanced-datascrubbing.mdx

```diff
@@ -75,7 +75,7 @@ Sentry does not know if a local variable that looks like a credit card number ac
 
 Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by event attribute, and can also be used to conservatively test rules on real data. A few examples:
 
-- `**` to scrub everything
+- `**` to scrub [almost every field](/product/data-management-settings/scrubbing/server-side-scrubbing/event-pii-fields/) (other fields require specific selectors)
```

Comment:
I would say 
```suggestion
- `**` to scrub [all currently supported fields](/product/data-management-settings/scrubbing/server-side-scrubbing/event-pii-fields/) (other fields require specific selectors)
```

Reply:
or maybe `all supported by default` fields?

Reply:
"all fields (that are) scrubbed by default" ?

Reply:
that's what the linked page says
<img width="878" alt="Screenshot 2024-03-13 at 9 53 35 AM" src="https://github.com/getsentry/sentry-docs/assets/490201/c4baf9c2-eb12-4711-add3-0da27f81843f">

Reply:
What about "all event PII fields"?

```suggestion
- `**` to scrub [all event PII fields](/product/data-management-settings/scrubbing/server-side-scrubbing/event-pii-fields/) (other fields require specific selectors)
```

Reply:
It might also be helpful to include a link to "other fields" if possible. I have no idea what those other fields are

Reply:
"all event PII fields" seems like other fields are not PII-scrubbable so I went with "all default event PII fields", let me know what you think.

We don't maintain a list of "other fields", the closest would be the event payloads in the [dev docs](https://develop.sentry.dev/sdk/event-payloads/) (with all the linked interfaces), but that's not friendly for the regular user. PII scrubbing is already a bit complex and requires some reading from the user, so I don't think adding another link with _a lot_ of content helps. Happy to add this link if somebody thinks it'd be helpful.

Reply:
I like "all default event PII fields" 👍 

I agree on not overloading the user with info. Instead of linking to other fields, are there any fields we can call out that users should be aware of? A somewhat common field they might expect to be scrubbed but isn't or an important field they might need to manually scrub? If not, that's ok and we can leave it as is.

Reply:
Added `span.description` as it has raised a couple of questions in the past, including this one.